ibis/backends/tests/test_register.py

```diff
@@ -107,7 +106,7 @@ def test_register_csv(con, data_dir, fname, in_table_name, out_table_name):
 
 
 # TODO: rewrite or delete test when register api is removed
-@pytest.mark.notimpl(["datafusion"])
+@pytest.mark.notimpl(["datafusion", "druid", "exasol", "oracle"])
```

Comment:
Some comment than above, why are we loosing this?

Reply:
same as above